app/services/exports/xml_export_service.rb

```diff
@@ -25,16 +25,16 @@ def build_export_run(collection, base_number, full_update)
       end
 
       if previous_exports_with_data.empty?
-        return Export.new(collection:, base_number:, started_at: @start_time)
+        return Export.new(collection:, year:, base_number:, started_at: @start_time)
       end
 
-      Export.new(collection:, started_at: @start_time, base_number:, increment_number:)
+      Export.new(collection:, year:, started_at: @start_time, base_number:, increment_number:)
     end
 
-    def write_export_archive(export, collection, recent_export, full_update)
-      archive = get_archive_name(collection, export.base_number, export.increment_number) # archive name would be the same for all logs because they're already filtered by year (?)
+    def write_export_archive(export, year, recent_export, full_update)
+      archive = get_archive_name(year, export.base_number, export.increment_number) # archive name would be the same for all logs because they're already filtered by year (?)
```

Comment:
Is the comment still useful?

Reply:
I don't understand what it means, so probably not